src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,47 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+
+/**
+ * Task with a date to complete the task by
+ */
+public class Deadline extends Task {
+    /**
+     * Deadline object by field which indicates the deadline
+     */
+    protected LocalDate by;
```

Comment:
Perhaps a name should explain the named entity to the reader accurately and at a sufficient level of detail?
```suggestion
    protected LocalDate byDate;
```

Reply:
Youre right, thank you!